docs/octant-plugin-installation.md

```diff
@@ -0,0 +1,108 @@
+# Octant and antrea-octant-plugin installation
+
+## Overview
+
+There are two ways to deploy Octant and antrea-octant-plugin.
+
+* Deploy Octant and antrea-octant-plugin as a Pod (inside Kubernetes cluster).
+
+* Deploy Octant and antrea-octant-plugin as a process (out of Kubernetes cluster).
```

Comment:
I guess we should not say "out of the K8s cluster" as the process could run on a Node too. If you want, we could say: could be out of K8s cluster.